swarmauri/standard/llms/concrete/DeepSeekModel.py

```diff
@@ -50,35 +48,17 @@ def predict(self,
         system_context = self._get_system_context(conversation.history)
```

Comment:
remove line 48 ` system_context = self._get_system_context(conversation.history)` and remove the `_get_system_context` function also